src/status_im/contexts/wallet/collectible/tabs/overview/view.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im.contexts.wallet.collectible.tabs.overview.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.collectible.tabs.overview.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- traits-section
+  [traits]
+  (when (pos? (count traits))
+    [rn/view
+     [quo/section-label
+      {:section         (i18n/label :t/traits)
+       :container-style style/traits-title-container}]
+     [rn/flat-list
+      {:render-fn               (fn [{:keys [trait-type value]}]
+                                  [quo/data-item
+                                   {:description     :default
+                                    :card?           true
+                                    :status          :default
+                                    :size            :default
+                                    :title           trait-type
+                                    :subtitle        value
+                                    :container-style style/traits-item}])
+       :data                    traits
+       :key                     :collectibles-list
+       :key-fn                  :id
+       :num-columns             2
+       :content-container-style style/traits-container}]]))
+
+(defn- info
+  [chain-id]
+  (let [network         (rf/sub [:wallet/network-details-by-chain-id
+                                 chain-id])
+        network-keyword (get network :network-name)
+        network-name    (string/capitalize (name network-keyword))]
+    [rn/view
+     {:style style/info-container}
+     [rn/view {:style style/account}
+      [quo/data-item
+       {:description         :account
+        :card?               true
+        :status              :default
+        :size                :default
+        :title               (i18n/label :t/account-title)
+        :subtitle            "Collectibles vault"
+        :emoji               "🎮"
+        :customization-color :yellow}]]
+
+     [rn/view {:style style/network}
+      [quo/data-item
+       {:description   :network
+        :card?         true
+        :status        :default
+        :size          :default
+        :title         (i18n/label :t/network)
+        :network-image (quo.resources/get-network network-keyword)
+        :subtitle      network-name}]]]))
+
+(defn- view-internal
+  []
+  (let [collectible-details (rf/sub [:wallet/last-collectible-details])
+        {:keys [traits id]} collectible-details
+        chain-id            (get-in id [:contract-id :chain-id])]
+    [:<>
+     [info chain-id]
```

Comment:
perhaps it's better to make more specific subs e.g `[:wallet/last-collectible-details-chain-id]` & `[:wallet/last-collectible-details-traits]`  and call these directly inside `info` and `traits-section` components respectively.

Cc @vkjr, @ulisesmac wdyt? 🤔

Reply:
I agree,

I can see we are subscribing to `:wallet/last-collectible-details`, extracting a parameter (`:chain-id`) in a child component we are subscribing to `:wallet/network-details-by-chain-id` passing the previous value we extracted.

This seems that, at least, we can create a single sub dependending on `:wallet/last-collectible-details` and use it inside the `info` component to simplify this code.

Reply:
Done. I created two new subs `[:wallet/last-collectible-details-chain-id]` and `[:wallet/last-collectible-details-traits]`